### PR TITLE
EPMRPP-35480 react stories for simple components

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/components/buttons/ghostButton/ghostButton.stories.jsx [hunk 1]
```diff
@@ -28,40 +28,32 @@ import { GhostButton } from './ghostButton';
...
-  .add('disabled with text', () => (
-    <GhostButton disabled>Button title</GhostButton>
-  ))
-  .add('disabled with icon', () => (
-    <GhostButton icon={TestIcon} disabled />
-  ))
+  .add('default state', () => <GhostButton />)
+  .add('with text', () => <GhostButton>Button title</GhostButton>)
+  .add('with icon', () => <GhostButton icon={TestIcon} />)
+  .add('with icon & text', () => <GhostButton icon={TestIcon}>Button title</GhostButton>)
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/buttons/ghostButton/ghostButton.stories.jsx]: Fixed story description (text meant children)

File: app/src/components/buttons/ghostMenuButton/ghostMenuButton.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+const items = [
+  {
+    label: 'Item label',
+    value: 'Item value',
+    onClick: action('clicked'),
+  },
+  {
+    label: 'Item label',
+    value: 'Item value',
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/buttons/ghostMenuButton/ghostMenuButton.stories.jsx]: Fixed

File: app/src/components/buttons/sidebarButton/sidebarButton.stories.jsx [hunk 3]
```diff
@@ -40,11 +39,6 @@ storiesOf('Components/Buttons/sidebarButton', module)
     }),
   )
   .addDecorator(withReadme(README))
-  /* TODO
-  .addDecorator(story => (
-    <MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>
-  ))
-  */
   .add('with icon', () => (
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/buttons/sidebarButton/sidebarButton.stories.jsx]: Done

File: app/src/components/buttons/toggleButton/README.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+Has no own size. Width and Height = 100% of it's container.
 
 ### Props:
-* **items**: _array_ of { value: _string_, label: _string_ }, optional, default = []
-* **value**: _string_, optional, default = ""
-* **onChange**: _func_, optional, default = () => {}
+
+- **items**: _array_ of { value: _string_, label: _string_ }, optional, default = []
+- **value**: _string_, optional, default = ""
+- **mobileDisabled**: _bool_, optional, default = false
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/buttons/toggleButton/README.md]: Fixed. Added stories to describe it

File: app/src/components/inputs/input/input.stories.jsx [hunk 5]
```diff
@@ -27,36 +27,30 @@ import { Input } from './input';
...
-  ))
-  .add('disabled', () => (
-    <Input disabled />
-  ))
+  .add('default state', () => <Input />)
+  .add('with placeholder', () => <Input placeholder="Placeholder test" />)
+  .add('with predefined value', () => <Input value="Predefined text" />)
+  .add('with predefined value & readonly', () => <Input readonly value="Predefined text" />)
+  .add('type number', () => <Input type="number" />)
+  .add('type password', () => <Input type="password" />)
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/inputs/input/input.stories.jsx]: Done

File: app/src/components/inputs/inputConditional/inputConditional.stories.jsx [hunk 6]
```diff
@@ -51,7 +51,7 @@ const conditions = [
 storiesOf('Components/Inputs/InputConditional', module)
   .addDecorator(
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/inputs/inputConditional/inputConditional.stories.jsx]: Done

File: app/src/components/inputs/inputConditionalTags/inputConditionalTags.stories.jsx [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+};
+
+const state = {
+  user: {
+    activeProject: 'superadmin_personal',
+  },
+};
+
+storiesOf('Components/Inputs/InputConditionalTags', module)
+  .addDecorator(
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/inputs/inputConditionalTags/inputConditionalTags.stories.jsx]: Done

File: app/src/components/buttons/bigButton/bigButton.stories.jsx [hunk 8]
```diff
@@ -26,32 +26,26 @@ import { withReadme } from 'storybook-readme';
...
-  .add('disabled', () => (
-    <BigButton disabled>Button title</BigButton>
-  ))
-  .add('with actions', () => (
-    <BigButton onClick={action('clicked')}>Button title</BigButton>
-  ))
+  .add('default state', () => <BigButton />)
+  .add('with text', () => <BigButton>Button title</BigButton>)
+  .add('with roundedCorners & text', () => <BigButton roundedCorners>Button title</BigButton>)
+  .add('colored', () => <BigButton color={'organish'}>Button title</BigButton>)
```
Comment (DiscoElevator) [inline app/src/components/buttons/bigButton/bigButton.stories.jsx]: `color="organish"`

File: app/src/components/inputs/input/README.md [hunk 9]
```diff
@@ -3,13 +3,23 @@
 Has no own size. Width and Height = 100% of it's parent.
```
Comment (DiscoElevator) [inline app/src/components/inputs/input/README.md]: That's not true anymore

File: app/src/components/inputs/inputDropdown/inputDropdown.jsx [hunk 10]
```diff
@@ -161,7 +161,8 @@ export class InputDropdown extends Component {
           multiple={this.props.multiple}
           subOption={!!option.groupRef}
           onChange={
-            !option.disabled && (option.groupId ? this.handleGroupChange : this.handleChange)
+            (!option.disabled && (option.groupId ? this.handleGroupChange : this.handleChange)) ||
+            (() => {})
```
Comment (DiscoElevator) [inline app/src/components/inputs/inputDropdown/inputDropdown.jsx]: null?

Comment (AmsterGet) [reply app/src/components/inputs/inputDropdown/inputDropdown.jsx]: Fixed

**Pull request merged**